src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    class="ec-summary-info"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-summary-info` : `ec-summary-info`"
+  >
+    <ec-icon
+      v-if="iconName"
+      class="ec-summary-info__main-icon"
+      data-test="ec-summary-info__main-icon"
+      :name="iconName"
+      :size="16"
+    />
+
+    <slot>
+      <div
+        data-test="ec-summary-info__content"
+        class="ec-summary-info__content"
+      >
+        <div
+          v-for="(item, index) in lineItems"
+          :key="item.text"
+          class="ec-summary-info__content-line-item"
+        >
+          <span
+            :title="item.text"
+            class="ec-summary-info__content-line-item-content"
+            :data-test="`ec-summary-info__content-line-item-content ec-summary-info__content-line-item-content-${getStylePreset(item.stylePreset, index)}`"
+            :class="`ec-summary-info__content-line-item-content-${getStylePreset(item.stylePreset, index)}`"
+          >
+            {{ item.text }}
+          </span>
+          <ec-icon
+            v-if="item.tooltipText"
+            v-ec-tooltip="{
+              content: item.tooltipText
+            }"
+            :class="`ec-summary-info__content-line-item-icon-${getStylePreset(item.stylePreset, index)}`"
+            :data-test="`ec-summary-info__content-line-item-icon ec-summary-info__content-line-item-icon-${getStylePreset(item.stylePreset, index)}`"
+            class="ec-summary-info__content-line-item-icon"
+            name="simple-info"
+            :size="14"
+          />
+        </div>
+
+      </div>
+    </slot>
+  </div>
+</template>
+
+<script setup>
+import VEcTooltip from '../../directives/ec-tooltip';
+import EcIcon from '../ec-icon';
+
+defineProps({
+  iconName: {
+    type: String,
+  },
+  lineItems: {
+    type: Array,
+    default: () => [],
+  },
+});
+
+function getStylePresetByIndex(index) {
+  if (index === 1 || index > 2) { return 'text'; }
+  if (index === 2) { return 'description'; }
+  return 'label';
+}
+
+function getStylePreset(stylePreset, index) {
+  if (stylePreset) {
+    return stylePreset;
+  }
+  return getStylePresetByIndex(index);
+}
+</script>
+
+<style>
+.ec-summary-info {
+  @apply tw-flex;
+
+  &__main-icon {
+    @apply tw-align-top;
+    @apply tw-flex-grow-0;
+    @apply tw-fill-gray-4;
+    @apply tw-mr-8;
+    @apply tw-flex-shrink-0;
+  }
+
+  &__content {
+    @apply tw-flex-grow;
```

Comment:
Hey @DisegnAle not sure, do we need those grow and width properties? Maybe is on a scenario I cannot think of.

I think though we do need overflow hidden here because ellipsis doesn't seem to work properly without it

```
  &__content {
    @apply tw-overflow-hidden;
  }

```